bids-validator/validators/json/schemas/microscopy.json

```diff
@@ -62,9 +62,9 @@
     "InstitutionName",
     "InstitutionAddress",
     "InstitutionalDepartmentName",
-
     "BodyPart",
-    "Environment",
+    "BodyPartDetails",
+    "SampleEnvironment",
     "SampleStaining",
     "SamplePrimaryAntibody",
     "SampleSecondaryAntibody"
```

Comment:
Just a question, why was `ChunkTransformationMatrix` removed from the list of recommended?
It is recommended if `chunk-<index>` is present in filenames (unless I made a mistake somewhere).
Just let me know, thanks!

Reply:
ChunkTransformationMatrix is removed from here because there's no way to check if chunk-<index> is present in the filenames in the microscopy schema. My solution is to check this in fullTest.js and if ChunkTransformationMatrix does not exist in the JSON file, there will be a warning raised.

Reply:
Perfect, thanks for the explanation!